web/templates/user_card_popover_content.hbs

```diff
@@ -129,28 +129,28 @@
             </li>
         {{/if}}
         {{#unless is_me}}
-        <li>
-            {{#if has_message_context}}
-            <a tabindex="0" class="mention_user">
-                <i class="fa fa-at" aria-hidden="true"></i>
-                {{#if is_bot}}{{t "Reply mentioning bot" }}{{else}}{{t "Reply mentioning user" }}{{/if}}
-                {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
-            </a>
-            {{else}}
-            <a tabindex="0" class="copy_mention_syntax" data-clipboard-text="{{ user_mention_syntax }}">
-                <i class="fa fa-at" aria-hidden="true"></i>
-                {{t "Copy mention syntax" }}
-                {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
-            </a>
-            {{/if}}
-        </li>
+            <li>
+                {{#if has_message_context}}
+                    <a tabindex="0" class="mention_user">
+                        <i class="fa fa-at" aria-hidden="true"></i>
+                        {{#if is_bot}}{{t "Reply mentioning bot" }}{{else}}{{t "Reply mentioning user" }}{{/if}}
+                        {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
+                    </a>
+                {{else}}
+                    <a tabindex="0" class="copy_mention_syntax" data-clipboard-text="{{ user_mention_syntax }}">
+                        <i class="fa fa-at" aria-hidden="true"></i>
+                        {{t "Copy mention syntax" }}
+                        {{#if is_sender_popover}}<span class="hotkey-hint">(@)</span>{{/if}}
+                    </a>
+                {{/if}}
+            </li>
         {{/unless}}
         {{#if is_me}}
-        <li>
-            <a href="/#settings/profile">
-                <i class="fa fa-edit" aria-hidden="true"></i> {{t "Edit your profile" }}
-            </a>
-        </li>
+            <li>
+                <a href="/#settings/profile">
+                    <i class="fa fa-edit" aria-hidden="true"></i> {{t "Edit your profile" }}
+                </a>
+            </li>
```

Comment:
It looks like your editor might have re-intended this template by accident? The indentation rules are intentional -- one doesn't need an extra indent for `if` statements, to help avoid what would otherwise be extreme indentation in some templates.